xml/en/docs/community.xml

```diff
@@ -0,0 +1,124 @@
+<!--
+  Copyright (C) F5, Inc.
+  -->
+
+<!DOCTYPE article SYSTEM "../../../dtd/article.dtd">
+
+<article name="Welcome to the NGINX Community!"
+         link="/en/docs/community.html"
+         lang="en"
+         rev="1">
+<section>
+
+<para>
+Open source powers much of the technology world. NGINX has been a strong part
```

Comment:
2 sentences in one line (Each new sentence starts with a new line), needs to be fixed globally